pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -827,7 +827,9 @@ protected void checkArgument(boolean b, String errorMessage) {
     }
 
     protected void validatePersistencePolicies(PersistencePolicies persistence) {
-        checkNotNull(persistence, "persistence policies should not be null");
+        if (persistence == null) {
+            return;
+        }
```

Comment:
Any reason to change the behavior here? If the request persistence policy is null, we should give a clear error message.

Reply:
In this way, `set` and `delete` can reuse the same method. set null is equal to delete. I will wrap a `Delete` method based on the `Set` method

Reply:
Yes, from the method definition, it has just done the request validation, so if we have a different purpose, we can use wrapper or others.